version-test/src/main/java/com/palantir/version/test/without/versions/VersionTestWithoutVersions.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.version.test.without.versions;
+
+public final class VersionTestWithoutVersions {
```

Comment:
Does Gradle run the tests against the jars or against the classes?

On Wed, Jun 22, 2016 at 2:36 PM jared2501 <notifications@github.com> wrote:

> In
> version-test/src/main/java/com/palantir/version/test/without/versions/VersionTestWithoutVersions.java
> <https://github.com/palantir/http-remoting/pull/140#discussion_r68139968>:
>
> > + * Licensed under the Apache License, Version 2.0 (the "License");
> > + * you may not use this file except in compliance with the License.
> > + * You may obtain a copy of the License at
> > + *
> > + *     http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +package com.palantir.version.test.without.versions;
> > +
> > +public final class VersionTestWithoutVersions {
>
> i decided not to since this might cause flakes if someone changes the
> build.gradle to include the name/version in the manifest (lol silly I
> know).. will change back though
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/140/files/09be5a3439b42390b2c70376e3eb41e41b168b7e#r68139968>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGOdwf6DGuPm73t1D0amEpFhubAzFSv2ks5qOarlgaJpZM4I7XZH>
> .
>